### PR TITLE
Upgrade OvS/OVN on EL9

### PR DESCRIPTION
**Pull request by mwperina**
EL9 versions (upgraded):
* OvS 2.17
* OVN 2022

EL8 version (untouched):
* OvS 2.15
* OVN 2021

Signed-off-by: Martin Perina <mperina@redhat.com>

### CODE REVIEW

File: SPEC/ovirt-openvswitch.spec [hunk 1]
```diff
@@ -1,12 +1,19 @@
+%if 0%{?rhel} >= 8
```
Comment (almusil) [inline SPEC/ovirt-openvswitch.spec]: Shouldn't this be just `>` 8?

Comment (mwperina) [reply SPEC/ovirt-openvswitch.spec]: Stupid me :-(

**Comment by michalskrivanek** [comment]
lgtm

File: SPEC/ovirt-openvswitch.spec [hunk 2]
```diff
@@ -51,18 +60,18 @@ Summary:        Wrapper for openvswitch-ipsec rpm
...
+Obsoletes:      openvswitch%{ovs_to_replace}-ipsec
 
 %description ipsec
 Wrapper rpm for the base openvswitch-ipsec package
 
 %package        ovn
 Summary:        Wrapper for ovn rpm
 License:        Public Domain
-Requires:       ovn-%{ovn_version}
+Requires:       ovn%{?ovn_dash}%{ovn_version}
```
Comment (almusil) [inline SPEC/ovirt-openvswitch.spec:72]: I don't see the `ovn_dash` definition I guess it should be included only on el9 right?

Comment (michalskrivanek) [reply SPEC/ovirt-openvswitch.spec:72]: yes. note the "?"

Comment (almusil) [reply SPEC/ovirt-openvswitch.spec:72]: Noted, but shouldn't it be defined for el9? That's my original question/

Comment (michalskrivanek) [reply SPEC/ovirt-openvswitch.spec:72]: ah, yeah (el8 actually). it was in my commit but i see martin replaced/removed that. sorry i didn't notice.

Comment (michalskrivanek) [reply SPEC/ovirt-openvswitch.spec:72]: @mwperina please fix that. we need ovn-XX for the old one and ovnYY for the new one, hence the optional ovn_dash

Comment (mwperina) [reply SPEC/ovirt-openvswitch.spec:72]: Hmm, I specifically search for `ovn_dash`  when merging commits and haven't found anything, no idea why.
Sorry, posted a fix

**Review by almusil — APPROVED** [review]

**Pull request merged**